src/utils/types.js

```diff
@@ -0,0 +1,10 @@
+const type = {
+  isArray: (value) => value && typeof value === 'object' && value.constructor === Array,
```

Comment:
The function `Array.isArray()` already does this:

```js
console.log(Array.isArray(2)); // false
console.log(Array.isArray([])); // true
```

Reply:
Ha, the article I was using for these checks even mentioned it as an option. Not sure why I didn't use it